src/Orderer.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+public class Orderer {
+
+    public Transaction[] orderTheTransactions(Transaction[] unorderedTransactions) {
```

Comment:
The design says that this should be called just `order` ('`theTransactions`' is implied through the input parameters).